README.adoc

```diff
@@ -11,40 +11,22 @@ See the link:https://docs.servicetalk.io/[ServiceTalk docs] for more information
 
 == Getting Started
 
-Currently, all ServiceTalk released artifacts are available in a Bintray repo:
-https://dl.bintray.com/servicetalk/servicetalk/, but will be published to Maven Central
-link:https://github.com/apple/servicetalk/issues/845[soon].
+ServiceTalk releases are available on Maven Central:
 
-This repository must be configured in which ever build tool your project uses (e.g. Maven, Gradle, etc...).
+* http://repo1.maven.org/maven2/io/servicetalk/
 
-.Maven, pom.xml
-[source,xml]
-----
-<project>
-...
-  <repositories>
-    <repository>
-      <id>bintray-servicetalk</id>
-      <url>https://dl.bintray.com/servicetalk/servicetalk/</url>
-    </repository>
-  </repositories>
-...
-</project>
-----
+For Gradle as well as other build tools that don't download from Maven Central by default, add it to your remote
```

Comment:
Consider rewording:
For Gradle as well as other build tools that don't use Maven Central as a default repository, additional configuration is required:

```
.Gradle, build.gradle
[source,groovy]
----
repositories {
  jcenter()  // combines Maven Central and other popular repositories
}
----
```